src/mounts.c

```diff
@@ -110,13 +111,18 @@ void mount_bind(char * source, char * dest, int writable) {
         ABORT(255);
     }
 
+    //  NOTE: This used to be separated into two distinct mount calls (one to mount, a second to remount
+    //  as read-only).  I changed this to a single mount() call as user-namespaces aren't allowed to
+    //  remount.
```

Comment:
I've been looking around, and it seems this is fairly kernel version dependent - with newer kernels skewing toward requiring two separate calls.